packages/deployments/contracts/contracts/messaging/connectors/polygon/PolygonHubConnector.sol

```diff
@@ -47,10 +47,8 @@ contract PolygonHubConnector is HubConnector, FxBaseRootTunnel {
     emit MessageProcessed(message, msg.sender);
   }
 
-  function _processMessage(bytes memory _data) internal override {
-    // Does nothing, all messages should go through the `_processMessageFromChild` path
-    revert Connector__processMessage_notUsed();
-  }
+  // DO NOT override _processMessage, should revert from `Connector` class. All messages must use the
+  // `processMessageFromRoot` flow.
```

Comment:
`processMessageFromRoot` needs to be `_processMessageFromChild` in the comment.

Reply:
fixed! https://github.com/connext/nxtp/pull/2493/commits/0dbd3f98b51a282474c8021d666cb9f70be7afb6